app_dart/test/request_handlers/check_for_waiting_pull_requests_test.dart

```diff
@@ -362,6 +387,34 @@ void main() {
       ]);
     });
 
+    test('Merge a clean revert PR with in progress tests', () async {
```

Comment:
Can we add a test for a non-clean revert?

Reply:
Please see `githubComparison = GitHubComparison('abc', 'def', 0, 0, 0, <CommitFile>[CommitFile(name: 'test')]);` in other tests, which is mocking the non-clean revert cases.